fpga.chart.py

```diff
@@ -179,32 +181,37 @@ def _get_xilinx_fpga_power(self, idx):
 
     def _get_fpga_temp(self, idx):
         if os.path.exists(self.intel_cmd) and os.access(self.intel_cmd, os.X_OK):
-            return self._get_intel_fpga_temp(idx)
+            self.temp[idx] = self._get_intel_fpga_temp(idx)
+            return
         if os.path.exists(self.xilinx_cmd) and os.access(self.xilinx_cmd, os.X_OK):
-            return self._get_xilinx_fpga_temp(idx)
+            self.temp[idx] = self._get_xilinx_fpga_temp(idx)
+            return
 
 
     def _get_fpga_power(self, idx):
         if os.path.exists(self.intel_cmd) and os.access(self.intel_cmd, os.X_OK):
-            return self._get_intel_fpga_power(idx)
+            self.power[idx] = self._get_intel_fpga_power(idx)
+            return
         if os.path.exists(self.xilinx_cmd) and os.access(self.xilinx_cmd, os.X_OK):
-            return self._get_xilinx_fpga_power(idx)
+            self.power[idx] = self._get_xilinx_fpga_power(idx)
+            return
 
 
     def get_fpga_os_status(self):
+        result_dict = {'result': '0'}
         while True:
-            time.sleep(self.temp_power_update_interval)
+            time.sleep(2)
```

Comment:
Why 2?

Reply:
That's been updated in the following push